cli/command/stack/cmd.go

```diff
@@ -1,50 +1,125 @@
 package stack
 
 import (
+	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	cliconfig "github.com/docker/cli/cli/config"
+	"github.com/docker/cli/cli/config/configfile"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
 var errUnsupportedAllOrchestrator = fmt.Errorf(`no orchestrator specified: use either "kubernetes" or "swarm"`)
 
+type commonOptions struct {
+	orchestrator command.Orchestrator
+}
+
 // NewStackCommand returns a cobra command for `stack` subcommands
 func NewStackCommand(dockerCli command.Cli) *cobra.Command {
+	var opts commonOptions
 	cmd := &cobra.Command{
-		Use:   "stack",
+		Use:   "stack [OPTIONS]",
```

Comment:
this output isn't actually used for commands that have subcommands, but good to have in case that situation changes in future